docs/Makefile

```diff
@@ -11,9 +11,12 @@
 
 # You can set these variables from the command line.
 #SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
 PAPER         =
 BUILDDIR      = build
+CFGPATH				= ../dirac.cfg
```

Comment:
Apologies, I proposed to put this in the Makefile, but I forgot that readthedocs doesn't use the makefile to build the documentation. It only uses the conf.py.
https://github.com/DIRACGrid/DIRAC/blob/fc835733a6f923e799664e96551310b31c1249e7/docs/source/conf.py#L49

Could you instead do the copy here for example?
https://github.com/DIRACGrid/DIRAC/blob/fc835733a6f923e799664e96551310b31c1249e7/docs/diracdoctools/cmd/concatcfg.py#L40

Reply:
This still needs to be moved from the makefile to the concatcfg command